docs/advanced_topics/api/v2/usage.md

```diff
@@ -0,0 +1,647 @@
+# Wagtail API v2 Usage Guide
+
+The Wagtail API module exposes a public, read only, JSON-formatted API which
+can be used by external clients (such as a mobile app) or the site's frontend.
+
+This document is intended for developers using the API exposed by Wagtail. For
+documentation on how to enable the API module in your Wagtail site, see
+[/advanced_topics/api/v2/configuration][1]
+
+[1]: https://github.com/wagtail/wagtail/blob/main/docs/advanced_topics/api/v2/configuration.md
+
+## Contents
+
+-   [Fetching content](#fetching-content)
+    -   [Example response](#example-response)
+    -   [Custom page fields in the API](#custom-page-fields-in-the-api)
+    -   [Pagination](#pagination)
+    -   [Ordering](#ordering)
+        -   [Random ordering](#random-ordering)
+    -   [Filtering](#filtering)
+    -   [Filtering by tree position (pages only)](#filtering-by-tree-position-pages-only)
+    -   [Search](#search)
+        -   [Search operator](#search-operator)
+    -   [Special filters for internationalised sites](#special-filters-for-internationalized-sites)
+        -   [Filtering pages by locale](#filtering-pages-by-locale)
+        -   [Getting translations of a page](#getting-translations-of-a-page)
+    -   [Fields](#fields)
+        -   [Additional fields](#additional-fields)
+        -   [All fields](#all-fields)
+        -   [Removing fields](#removing-fields)
+        -   [Removing all default fields](#removing-all-default-fields)
+    -   [Detail views](#detail-views)
+    -   [Finding pages by HTML path](#finding-pages-by-html-path)
+-   [Default endpoint fields](#default-endpoint-fields)
+    -   [Common fields](#common-fields)
+    -   [Pages](#pages)
+    -   [Images](#images)
+    -   [Documents](#documents)
+-   [Changes since v1](#changes-since-v1)
+    -   [Breaking changes](#breaking-changes)
+    -   [Major features](#major-features)
+    -   [Minor features](#minor-features)
+
+---
+
+## Fetching content
+
+To fetch content over the API, perform a `GET` request against one of the
+following endpoints:
+
+-   Pages `/api/v2/pages/`
+-   Images `/api/v2/images/`
+-   Documents `/api/v2/documents/`
+
+> The available endpoints and their URLs may vary from site to site, depending
+> on how the API has been configured.
+
```

Comment:
Not sure on the caret syntax here, maybe just make it plain text and remove `>`

Reply:
`>` would yield a blockquote. so a callout alternative?

Reply:
I added [{note} syntax](https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#note-and-warning-call-outs) to match the current docs